<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>evoKGsim+: a framework for tailoring Knowledge Graph-based similarity for supervised learning</title>
  <meta content="" name="description">
  <meta content="" name="keywords">

  <!-- Favicons -->
  <link href="assets/img/favicon.png" rel="icon">
  <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon">

  <!-- Google Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Raleway:300,300i,400,400i,500,500i,600,600i,700,700i|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i" rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/aos/aos.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href="assets/vendor/boxicons/css/boxicons.min.css" rel="stylesheet">
  <link href="assets/vendor/glightbox/css/glightbox.min.css" rel="stylesheet">
  <link href="assets/vendor/remixicon/remixicon.css" rel="stylesheet">
  <link href="assets/vendor/swiper/swiper-bundle.min.css" rel="stylesheet">

  <!-- Template Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">

  <!-- =======================================================
  * Template Name: OnePage - v4.2.0
  * Template URL: https://bootstrapmade.com/onepage-multipurpose-bootstrap-template/
  * Author: BootstrapMade.com
  * License: https://bootstrapmade.com/license/
  ======================================================== -->
</head>

<body>



<!-- ======= Header ======= -->
  <header id="header" class="fixed-top">
    <div class="container d-flex align-items-center justify-content-between">

      <h1 class="logo"><a href="index.html">evoKGsim+</a></h1>
      <!-- Uncomment below if you prefer to use an image logo -->
      <!-- <a href="index.html" class="logo"><img src="assets/img/logo.png" alt="" class="img-fluid"></a>-->

      <nav id="navbar" class="navbar">
        <ul>

          <li><a class="nav-link scrollto active" href="#hero">Home</a></li>
          <li><a class="nav-link scrollto" href="#motivation">Motivation</a></li>
          <li><a class="nav-link scrollto" href="#methodology">Methodology</a></li>
          <li class="dropdown"><a href="#strategy"><span>Evaluation</span> <i class="bi bi-chevron-down"></i></a>
            <ul>
              <li><a href="https://github.com/liseda-lab/kgsim-benchmark">Data</a></li>
              <li><a href="#strategy">Evaluation Strategy</a></li>
              <li><a href="#results">Results</a></li>
            </ul>
          </li>
          <li><a class="nav-link scrollto" href="#conclusions">Conclusions</a></li>
          <li><a class="nav-link scrollto" href="#minute-video">One-Minute Video</a></li>
          <li><a class="nav-link scrollto" href="#authors">Authors</a></li>
          <li><a class="nav-link scrollto" href="#funding">Funding</a></li
          <li><a class="getstarted scrollto" href="https://openreview.net/pdf/c1c08929d05e8424a472d8b1e5d837a02e5ea6ae.pdf">Short Paper</a></li>

        </ul>
        <i class="bi bi-list mobile-nav-toggle"></i>
      </nav><!-- .navbar -->

    </div>
  </header><!-- End Header -->



<!-- ======= Hero Section ======= -->
  <section id="hero" class="d-flex align-items-center">
    <div class="container position-relative" data-aos="fade-up" data-aos-delay="100">

      <div class="row justify-content-center">
        <div class="col-xl-9 col-lg-12 text-center">
          <h1>evoKGsim+</h1>
            <h3>A framework for tailoring Knowledge Graph-based similarity for supervised learning</h3>
          <h2>Rita T. Sousa, Sara Silva, Catia Pesquita</h2>
          <p>LASIGE, Faculdade de CiÃªncias, Universidade de Lisboa</p>
        </div>
      </div>

      <div class="text-center">
        <a href="https://github.com/liseda-lab/evoKGsim" class="btn-get-started scrollto"> GitHub</a>
      </div>

      <div class="row icon-boxes">

        <div class="col-md-6 col-lg-3 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="zoom-in" data-aos-delay="200">
          <div class="icon-box">
            <div class="icon"><i class="bi bi-share-fill"></i></div>
            <h4 class="title"> When predicting external relations between KG entities... </h4>
            <p class="description">A classification task that takes as input a KG and a set of KG entity pairs for which we want to predict a relation that is not encoded in the KG.</p>
          </div>
        </div>

        <div class="col-md-6 col-lg-3 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="zoom-in" data-aos-delay="300">
          <div class="icon-box">
            <div class="icon"><i class="bi bi-diagram-3"></i></div>
            <h4 class="title"> ... not everything in the KG will be useful.</h4>
            <p class="description">A large portion of a general-purpose KG may be irrelevant for a specific prediction task. </p>
          </div>
        </div>

        <div class="col-md-6 col-lg-3 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="zoom-in" data-aos-delay="400">
          <div class="icon-box">
            <div class="icon"><i class="ri-command-line"></i></div>
            <h4 class="title">In complex domains...</h4>
            <p class="description">KGs can represent multiple views (or semantic aspects) over the underlying data and some may be less 
              relevant to train the model towards a specific target. </p>
          </div>
        </div>

        <div class="col-md-6 col-lg-3 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="zoom-in" data-aos-delay="500">
          <div class="icon-box">
            <div class="icon"><i class="bi bi-scissors"></i></div>
            <h4 class="title">... we need to tailor semantic representations</h4>
            <p class="description">of the KG entities to an independent and specific classification task when the classification target is not encoded in the KG.</p>
          </div>
        </div>

      </div>

    </div>
  </section><!-- End Hero -->

<main id="main">



<!-- ======= Counts Section ======= -->
  <section id="counts" class="counts section-bg">
      <div class="container">

        <div class="row justify-content-end">

          <div class="col-lg-4 col-md-5 col-6 d-md-flex align-items-md-stretch">
            <div class="count-box">
              <span data-purecounter-start="0" data-purecounter-end="6" data-purecounter-duration="2" class="purecounter"></span>
              <h4>Taxonomic Similarity Measures</h4>
            </div>
          </div>

          <div class="col-lg-4 col-md-5 col-6 d-md-flex align-items-md-stretch">
            <div class="count-box">
              <span data-purecounter-start="0" data-purecounter-end="4" data-purecounter-duration="2" class="purecounter"></span>
              <h4>KG Embeddings Methods</h4>
            </div>
          </div>

          <div class="col-lg-4 col-md-5 col-6 d-md-flex align-items-md-stretch">
            <div class="count-box">
              <span data-purecounter-start="0" data-purecounter-end="10" data-purecounter-duration="2" class="purecounter"></span>
              <h4>Benchmark datasets</h4>
            </div>
          </div>

        </div>

      </div>
  </section><!-- End Counts Section -->



<!-- ======= About Section ======= -->
  <section id="motivation" class="about">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Motivation </h2>
        </div>

        <div class="row content">

          <div class="col-lg-6">
            <p>
            <a href="https://dl.acm.org/doi/pdf/10.1145/3227609.3227689?casa_token=-4vHP2XEhWYAAAAA:AxYIsd1j4X8I4mzlx5qovlcuVN-nW_wEBEWqOqxzyIdeQqt7o0l86R18B66UnQ1AjixEUl-5hSlWSQ" > KGs</a>
            have been explored as providers of features and background knowledge in several machine learning application scenarios, given their ability to provide meaningful context to the data through 
            semantic representations. Despite the <a href="https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=8047276&casa_token=sQR3EFlY4K0AAAAA:PwH2C0NEeLIscTmflgMc31tyuaJrPD3d62Z0IXc9aDm5-D-40ed5GHWxhfJ8_TvkYcLE6wsn8Co&tag=1" > recent 
            advances in semantic representations</a>, existing methods are unsuited to tailoring semantic representations to a 
            specific learning target that is not encoded in the KG. 
            </p>
            <a data-bs-toggle="collapse" class="collapsed" data-bs-target="#problem"> Learn more <i class="bx bx-chevron-down icon-show"></i><i class="bx bx-chevron-up icon-close"></i></a>
            <div id="problem" class="collapse">
              This problem is exacerbated in complex domains, such as the biomedical, where KGs represent multiple views (or semantic aspects) 
              over the underlying data, some of which may be less relevant to train the model towards a specific target. 
              For instance, the prediction of protein-protein interactions using the <a href="http://geneontology.org/"  > Gene Ontology</a> is more accurate if only a portion of 
              the ontology is used (in this case, the one concerning biological processes).
            </div>
          </div>

          <div class="col-lg-6 pt-4 pt-lg-0">
            <p>
            <a href="https://openreview.net/pdf/c1c08929d05e8424a472d8b1e5d837a02e5ea6ae.pdf" > evoKGsim+ </a> is a framework that learns suitable semantic similarity-based semantic representations of data objects 
            extracted from KGs optimized for supervised learning. This tailoring is achieved by evolving a suitable combination 
            of semantic aspects using <a href="https://digitalcommons.morris.umn.edu/cgi/viewcontent.cgi?referer=https://scholar.google.com/&httpsredir=1&article=1001&context=cs_facpubs" > genetic programming</a> (GP). 
            The evolution of the semantic representations is guided by a fitness function based on the success of a given semantic representation in a specific task.
            </p>
            <a data-bs-toggle="collapse" class="collapsed" data-bs-target="#evokgsim"> Learn more <i class="bx bx-chevron-down icon-show"></i><i class="bx bx-chevron-up icon-close"></i></a>
            <div id="evokgsim" class="collapse">
              Previously, we presented <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3296-1.pdf" > evoKGsim</a>, but it is 
              limited to taxonomy-based semantic similarity measures, which do not consider the full spectrum of semantic relations that can be established within a KG.  
              evoKGsim+ is an extension of evoKGsim into a full framework that combines several taxonomic and embedding similarity measures and provides several baseline 
              evaluation approaches that emulate domain expert feature selection and optimal parameter setting.
            </div>
          </div>

        </div>

      </div>
  </section><!-- End About Section -->



<!-- ======= Methodology Section ======= -->
  <section id="methodology" class="methodology">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Methodology</h2>
          <div class="row justify-content-center">
            <div class="col-8" >
              <span> evoKGsim+, available on <a href="https://github.com/liseda-lab/evoKGsim" > GitHub </a>, targets classification tasks that take as input a KG and a set of KG individual pairs for which we wish to learn a relation that is outside the scope of the KG. </span>
            </div>
          </div>
        </div>

        <div class="row">

          <div class="col-lg-4 col-md-6" data-aos="zoom-in" data-aos-delay="100">
            <div class="icon-box iconbox-blue">
              <h4><a href="">Step I</a></h4>
              <p>Represent each instance (i.e., a pair of KG entities) according to KG-based similarities computed for each semantic aspect. </p>
              <a data-bs-toggle="collapse" class="collapsed" data-bs-target="#step1"> More info <i class="bx bx-chevron-down icon-show"></i><i class="bx bx-chevron-up icon-close"></i> </a> 
                <div id="step1" class="collapse">
                  <p> evoKGsim+ currently supports 10 different KG-based similarity measures based on a selection of representative state-of-the-art approaches: </p>
                  <p>
                  <i class="ri-check-double-line"></i> 6 taxonomic semantic similarity measures, derived by combining one of two information content approaches 
                  (<a href="https://www.researchgate.net/profile/Jer-Hayes/publication/220837848_An_Intrinsic_Information_Content_Metric_for_Semantic_Similarity_in_WordNet/links/0fcfd50a2122fa6910000000/An-Intrinsic-Information-Content-Metric-for-Semantic-Similarity-in-WordNet.pdf" > ICSeco </a> 
                  and <a href="http://www.cs.cmu.edu/afs/cs.cmu.edu/project/jair/pub/volume11/resnik99a.pdf" > ICResnik </a>) with one of three set similarity measures (<a href="http://www.cs.cmu.edu/afs/cs.cmu.edu/project/jair/pub/volume11/resnik99a.pdf" > ResnikMax </a>, 
                  <a href="http://www.cs.cmu.edu/afs/cs.cmu.edu/project/jair/pub/volume11/resnik99a.pdf" > ResnikBMA </a>, and <a href="https://webpages.ciencias.ulisboa.pt/~fjcouto/files/workshop%20cpesquita-ismb2007.pdf" > SimGIC </a>). 
                  </p>
                  <p>
                  <i class="ri-check-double-line"></i> 4 measures based on cosine similarity over embeddings generated from 
                  <a href="https://hal.archives-ouvertes.fr/file/index/docid/920777/filename/bordes13nips.pdf" > TransE </a>, <a href="https://arxiv.org/pdf/1412.6575.pdf" > distMult </a>, 
                  <a href="https://madoc.bib.uni-mannheim.de/41307/1/Ristoski_RDF2Vec.pdf" > RDF2Vec </a> and <a href="https://core.ac.uk/download/pdf/340075163.pdf" > Owl2Vec </a>.
                  </p>
                </div>
            </div>
          </div>

          <div class="col-lg-4 col-md-6 mt-4 mt-lg-0" data-aos="zoom-in" data-aos-delay="200">
            <div class="icon-box iconbox-orange ">
              <h4><a href="">Step II</a></h4>
              <p> Employ GP to learn a suitable combination of the different aspect-based similarities, using a set of predefined operators, to address a given task. </p>
              <a data-bs-toggle="collapse" class="collapsed" data-bs-target="#step2"> More info <i class="bx bx-chevron-down icon-show"></i><i class="bx bx-chevron-up icon-close"></i> </a> 
                <div id="step2" class="collapse">
                  <p> <a href="http://geneticprogramming.com/" > GP</a> is a population-based search procedure inspired in Darwinian evolution and Mendelian genetics.</p>
                  <p>
                  GP starts by creating an initial population of random individuals representing potential combination of semantic aspects. 
                  Each individual solution is evaluated and assigned a fitness value that quantifies how accurate are the classification predictions 
                  made by using this combination. New generations of potential combinations are iteratively created by selecting parents based on their 
                  fitness and breeding them using genetic operators like crossover and mutation. The fitter individuals are selected more often to be parents 
                  and thus pass their characteristics to their offspring, which causes the population to improve in quality along successive generations.
                  </p> 
                  <iframe width="420" height="315"
                  src="https://www.youtube.com/embed/cxweR4i0ejA">
                  </iframe>
                  <p> Learn more <a href="https://machinelearningmastery.com/books-on-genetic-programming/" > here </a>.</p>
                </div>
            </div>
          </div>

          <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4 mt-lg-0" data-aos="zoom-in" data-aos-delay="300">
            <div class="icon-box iconbox-pink">
              <h4><a href="">Step III</a></h4>
              <p>Evaluate the predictions made on the test set and comparing them against optimized static representations that represent expert feature selection and parameter tuning. </p>
              <a data-bs-toggle="collapse" class="collapsed" data-bs-target="#step3"> More info <i class="bx bx-chevron-down icon-show"></i><i class="bx bx-chevron-up icon-close"></i> </a> 
                <div id="step3" class="collapse">
                  <p>
                  GP is used in a regression-like fashion, treating the expected class labels (0 and 1) as numeric expected outputs, and calculating fitness as the root mean squared error between the 
                  expected and predicted values, a normal procedure when using GP for binary classification. 
                  </p>
                  <p>
                  The predicted numeric outputs are transformed in class labels, by applying the natural cutoff of 0.5, only for reporting performance.
                  </p>
                </div>
            </div>
          </div>

        </div>

        <div class="row justify-content-center">
          <div class="mxgraph" style="max-width:100%;border:1px solid transparent;" data-mxgraph="{&quot;highlight&quot;:&quot;#0000ff&quot;,&quot;nav&quot;:true,&quot;zoom&quot;:6,&quot;resize&quot;:true,&quot;toolbar&quot;:&quot;zoom layers lightbox&quot;,&quot;edit&quot;:&quot;_blank&quot;,&quot;xml&quot;:&quot;&lt;mxfile modified=\&quot;2021-05-26T22:35:52.447Z\&quot; host=\&quot;app.diagrams.net\&quot; agent=\&quot;5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36\&quot; etag=\&quot;LzheYMpYLJdywMoMNkiF\&quot; version=\&quot;14.7.2\&quot; type=\&quot;google\&quot;&gt;&lt;diagram name=\&quot;Page-1\&quot; id=\&quot;e7e014a7-5840-1c2e-5031-d8a46d1fe8dd\&quot;&gt;7VxZc+I4EP41VO0+kPJteJyQZI6dI1PM7Cb7JowwnhiLkUWA/fUr2bItWcYYMARmQqWCdUvdXx9qd9IxB7PVWwzm009oDMOOoY1XHfOmYxi6ZRgd9qON12mNq9lphY+DMe9UVAyD/yCv1HjtIhjDWOpIEApJMJcrPRRF0CNSHcAYLeVuExTKq86BD5WKoQdCtfafYEymvFZ3+kXDOxj4U750z3DShhHwnnyMFhFfr2OYk+STNs9ANhc/aDwFY7QUqszbjjnACJH0abYawJDRNiNbOu5uQyvfd0zW2Umy/WIYkYoZ0OgHo5+hhWBEeZh0GKDZfEEACVBEG9CETRjMghDggKwTamL6GwJvylrgDEQk8Bjh43kylyFsKB5o3z7YPx/C4Tv9w+h5+P395N+upecnzU8gbDkhH2QTaB3zejkNCBzOgcdalxRutG5KZmyvOn3MCcgKHprRnaTj/BDEMX9WicAJ9QwxgSuhim/qLUQzSDA7LW81XU5Kjmg94+BSwIfG66YCNBxeBzgk/Xzqggv0gRMiK3K+JMXvMcRfKvjUMZyQdrmeoIhIFHR+LlDW0I0T4XpDO+j6fFU00iefff8VoWUIx1QeDC2R5mxauot05rSfJIUitupZbdWymnJvzh69dRhQnuPt/B6l4Pg4yityifuyIHQWWAWMFgBAZVwGgKkCwKngv70X/wWeV0t6NbW5ln0G4UKSf4HkMBq/YRqSlkYh8p5k8hayl9CQYPQEByikEs8Gm1ryoS10lrsgDLeRN8FVHXEF4tk1xMMwpProWdbOVRTlK9yjIJGJTHitEu9sS54iRgvsQT5K1K/licpaoDwRAdiHRJmIUhyshW5z1iHefcMFXtIZC/TkNK0CVDP0ZMTeHz1wFZAHpnOv7L7Dy4+s7UrTXF6+WXGlnBTWvMDkMKD2900Y+BGtI4jJPYPPkC+OMJkiH0UgvC1qr2VbIaNyQh8F7EJbt7Rs4XuIA0ovpnH4SEq7B7HwmJ4jKxbbTkrZvjciP0XUViNINWqCmDq2GGclS5YmQ5NW7CdLdnmisqZsSZYss7SOI3lAR5IlQ5GlT9RhYuaJ2nAIcBREfmJPEwgEYeJTpR5UF+GAwomiWnKvJB9sBkG8wDBWBHR/56kFG2lnbnBG6ionSa/AZm93I9mQD1UWsQ236RsGAefhEJJan+ky/R3b1k/l7zRkpXuoeapzZzL1X2j8R0nhV6t/weCJ5q7W1DV0nbYakAzal2ZAbKukJLR9DUhfnkhvaEBa0/H9QwF5DK9nX09mk7e0EaBbceec1yXAyRyVTf5GU9y5pnkSx8VxepXrHNVxyZgmgPqWPfNw0Dl5G06/7G3YioVyT2mhHKuBQthRuIs+HxHTDgnxfkBC1jxqChbUh5RvYarOkVXEZju4vzVyjLa1QmNxP4xnqqPekoMI49wtHOHCI3TAjMlFNIrZ16/pO7ramfmOjnOOpnpnk3smltTNXoMcbEn7WyZqyZK6/WrT35YlvV9//n7/8BXfPkyJHhgfRjd9v6sa0h0hV1wuLC0r82iaft7RtP3NiHVeSC/Hew2tf9V39wO7pctzmfqhYD/gQlOJ2CZvD07hvlTfxfO2DReYmNKPZOIUIWbcTnF5OViXlhBWDoW1pQJN2SSbVo0KVK/aPXm00adyIH4aiURbOFVdtgGKqFO78KR31uwVtpdwaI5hTBGQvdIegRgy76fN6OkJXz3niuMIr54VBtw8kslb+4aK09r+jP9+P3A+rrpG76I1hWB2t+kJ0WDVYfHYamJHAdzblKhWqwy+hq97tst6w7QGL/c5ihtWnlNTvnQNCZzTMe9r71RU9J/SpcaAAHqHA1xv3P2I2deg414XU6eNsbAWvfTZ11IfgnyfQlrqIrZ79NDqDHm5UsZ0ob9903GTn/QhtWBiJkZxbmOb7PZMAWVtJd9onF9STMG9u7tTog1c4FhnLtd6OfrIu5Sd2BEiBM3aUam5lc0uqFqFSu33VJVq9HfXqRXpHMfE/q8AfsfcB/2N4F+fkPR7wN9qAH+jyqO4BPj/Cvg33b3w30gA7FcBMJwXFoBmbnZPjVreQzxBeAYijyWSfKIrsytOK7cYbROvAGeAx5JScDsM6GmliJxpXKlvb3oVHDDtnRlQleS8g8dUn63M4nZ5wqRKmSynLoiDUTakfPMWk32PnZBdfZ87kkrYdClupirEFpWwm1n1mtbdaTGtu9o+1QLmFG+qzgtGZ5YcXn0oNV3lZcNml6khXj765pgKI0s3oBbNU8MtqZkP55zuqro6IpI2Ya8hwnL2XEbabPUZfr+02XPDxOnSb+vc40ZXStM9pYfcDMHuCfLYjmqoTHdXyLxkPlw1D4xjaZFLzK06M7S8ZI5WNVpUtyZjXzwH0Wa0TAD1FNYpXnIM8A5TGD5DFkYqN7C+ye8y4AwGOKktXZU1Riz0EsrNS04b1s4yYrLGEBICcZdu3WP2rmo85RTpMrxFfLwmrJ00EgyimAV8svGJQ5V2WCI8lqcXhxdw7PIwKuuB/dEfBvvDAY1FOg2t9PxnPnwcxPMQcKIGURgIC09CBIi4obJ4SkGqXABTJuaimgvvK1ubsLVM41fRuEjRyCK2G6SiZJbYQbebIiU+e6CzurstUaO5qi1pKZZLi8V/x0iTDIp/QWLe/g8=&lt;/diagram&gt;&lt;/mxfile&gt;&quot;}"></div>
          <script type="text/javascript" src="https://viewer.diagrams.net/js/viewer-static.min.js"></script>
        </div>

      </div>
  </section><!-- End Methodology Section -->



<!-- ======= Evaluation Section ======= -->
  <section id="strategy" class="services section-bg">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Evaluation Strategy</h2>
          <div class="row justify-content-center">
            <div class="col-8" >
              <span> We evaluate the framework in its full extension in protein-protein interaction (PPI) prediction using the <a href="https://academic.oup.com/nar/article/49/D1/D325/6027811?login=true" > Gene Ontology</a> (GO) 
               as the KG and a set of <a href="https://academic.oup.com/database/article/doi/10.1093/database/baaa078/5979744?login=true"> benchmark data sets</a> with a wide range in size and semantic annotation characteristics. 
              </span>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="col-lg-6" data-aos="zoom-in" data-aos-delay="100">
            <div class="icon-box iconbox-blue">
              <div class="icon">
                <svg width="100" height="100" viewBox="0 0 600 600" xmlns="http://www.w3.org/2000/svg">
                  <path stroke="none" stroke-width="0" fill="#f5f5f5" d="M300,521.0016835830174C376.1290562159157,517.8887921683347,466.0731472004068,529.7835943286574,510.70327084640275,468.03025145048787C554.3714126377745,407.6079735673963,508.03601936045806,328.9844924480964,491.2728898941984,256.3432110539036C474.5976632858925,184.082847569629,479.9380746630129,96.60480741107993,416.23090153303,58.64404602377083C348.86323505073057,18.502131276798302,261.93793281208167,40.57373210992963,193.5410806939664,78.93577620505333C130.42746243093433,114.334589627462,98.30271207620316,179.96522072025542,76.75703585869454,249.04625023123273C51.97151888228291,328.5150500222984,13.704378332031375,421.85034740162234,66.52175969318436,486.19268352777647C119.04800174914682,550.1803526380478,217.28368757567262,524.383925680826,300,521.0016835830174"></path>
                </svg>
                <i class="bi bi-diagram-3"></i>
              </div>
              <h4><a href="">Gene Ontology Knowledge Graph</a></h4>
                <p>
                <a href="http://geneontology.org/" > GO</a> describes protein function with respect to three semantic aspects: biological process (BP), cellular component (CC), and molecular function (MF). 
                GO and the annotations that link proteins to GO classes make up a KG.
                </p>
                <p> <a data-bs-toggle="collapse" class="collapsed" data-bs-target="#GOKG"> See GO KG <i class="bx bx-chevron-down icon-show"></i><i class="bx bx-chevron-up icon-close"></i></a> </p>
                <div id="GOKG" class="collapse">
                  <img src="assets/img/GO.png" class="img-fluid" alt="" style="margin-right:20px">
                </div>
            </div>
          </div>

          <div class="col-lg-6" data-aos="zoom-in" data-aos-delay="200">
            <div class="icon-box iconbox-orange ">
              <div class="icon">
                <svg width="100" height="100" viewBox="0 0 600 600" xmlns="http://www.w3.org/2000/svg">
                  <path stroke="none" stroke-width="0" fill="#f5f5f5" d="M300,582.0697525312426C382.5290701553225,586.8405444964366,449.9789794690241,525.3245884688669,502.5850820975895,461.55621195738473C556.606425686781,396.0723002908107,615.8543463187945,314.28637112970534,586.6730223649479,234.56875336149918C558.9533121215079,158.8439757836574,454.9685369536778,164.00468322053177,381.49747125262974,130.76875717737553C312.15926192815925,99.40240125094834,248.97055460311594,18.661163978235184,179.8680185752513,50.54337015887873C110.5421016452524,82.52863877960104,119.82277516462835,180.83849132639028,109.12597500060166,256.43424936330496C100.08760227029461,320.3096726198365,92.17705696193138,384.0621239912766,124.79988738764834,439.7174275375508C164.83382741302287,508.01625554203684,220.96474134820875,577.5009287672846,300,582.0697525312426"></path>
                </svg>
                <i class="bi bi-share-fill"></i>
              </div>
              <h4><a href="">Protein-Protein Interaction Datasets</a></h4>
              <p> <a href="https://link.springer.com/content/pdf/10.1007/s11704-019-8232-z.pdf" > PPI prediction </a> was chosen as evaluation domain since it is well known that BP and CC aspects describe properties that are stronger indicators for PPI than the MF aspect for protein interaction.</p>
              <p> <a data-bs-toggle="collapse" class="collapsed" data-bs-target="#PPIdatasets"> See the number of interactions of each PPI dataset  <i class="bx bx-chevron-down icon-show"></i><i class="bx bx-chevron-up icon-close"></i></a> </p>
              <div id="PPIdatasets" class="collapse">
                <p> The <a href="https://academic.oup.com/database/article/doi/10.1093/database/baaa078/5979744?login=true"> PPI Benchmark datasets</a> are available on <a href="https://github.com/liseda-lab/kgsim-benchmark">GitHub</a>. </p> 
                <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
                <script type="text/javascript">
                google.charts.load('current', {'packages':['table']});
                google.charts.setOnLoadCallback(drawTable);
                function drawTable() {
                var data = new google.visualization.DataTable();
                data.addColumn('string', 'Benchmark Dataset');
                data.addColumn('string', 'Annotation Level');
                data.addColumn('number', 'Number of Proteins');
                data.addColumn('number', 'Number of Pairs');
                data.addRows([
                ['ALL1', 'incomplete', 11695, 59822],
                ['ALL3', 'complete', 10156, 47196],
                ['HS1', 'incomplete', 7093 , 30826],
                ['HS3', 'complete', 6718, 29672],
                ['SC1', 'incomplete', 3776 , 27898],
                ['SC3', 'complete', 2888, 16904],
                ['DM1', 'incomplete', 455, 364],
                ['DM3', 'complete', 287, 200],
                ['EC1', 'incomplete', 371, 734],
                ['EC3', 'complete', 263, 420]
                ]);
                var table = new google.visualization.Table(document.getElementById('table_results'));
                table.draw(data, {showRowNumber: false, width: '100%', height: '100%'});}
                </script>
                <div id="table_results"></div>
              </div>
            </div>
          </div>

        </div>

        <div class="row mt-4"></div>

        <div class="row">
          <div class="col-lg-6" data-aos="zoom-in" data-aos-delay="300">
            <div class="icon-box iconbox-pink">
              <div class="icon">
                <svg width="100" height="100" viewBox="0 0 600 600" xmlns="http://www.w3.org/2000/svg">
                  <path stroke="none" stroke-width="0" fill="#f5f5f5" d="M300,541.5067337569781C382.14930387511276,545.0595476570109,479.8736841581634,548.3450877840088,526.4010558755058,480.5488172755941C571.5218469581645,414.80211281144784,517.5187510058486,332.0715597781072,496.52539010469104,255.14436215662573C477.37192572678356,184.95920475031193,473.57363656557914,105.61284051026155,413.0603344069578,65.22779650032875C343.27470386102294,18.654635553484475,251.2091493199835,5.337323636656869,175.0934190732945,40.62881213300186C97.87086631185822,76.43348514350839,51.98124368387456,156.15599469081315,36.44837278890362,239.84606092416172C21.716077023791087,319.22268207091537,43.775223500013084,401.1760424656574,96.891909868211,461.97329694683043C147.22146801428983,519.5804099606455,223.5754009179313,538.201503339737,300,541.5067337569781"></path>
                </svg>
                <i class="bi bi-arrows-collapse"></i>
              </div>
              <h4><a href="">Static Semantic Representations</a></h4>
              <p> Five static representations as baselines: the BP, CC and MF single aspectsand the average and maximum of the single aspects. </p>
              <p>
              The static representations are employed as a simple similarity threshold-based classifier, where a semantic similarity score for a protein pair exceeding a certain threshold predicts a positive interaction.
              </p>
            </div>
          </div>

          <div class="col-lg-6" data-aos="zoom-in" data-aos-delay="100">
            <div class="icon-box iconbox-yellow">
              <div class="icon">
                <svg width="100" height="100" viewBox="0 0 600 600" xmlns="http://www.w3.org/2000/svg">
                  <path stroke="none" stroke-width="0" fill="#f5f5f5" d="M300,503.46388370962813C374.79870501325706,506.71871716319447,464.8034551963731,527.1746412648533,510.4981551193396,467.86667711651364C555.9287308511215,408.9015244558933,512.6030010748507,327.5744911775523,490.211057578863,256.5855673507754C471.097692560561,195.9906835881958,447.69079081568157,138.11976852964426,395.19560036434837,102.3242989838813C329.3053358748298,57.3949838291264,248.02791733380457,8.279543830951368,175.87071277845988,42.242879143198664C103.41431057327972,76.34704239035025,93.79494320519305,170.9812938413882,81.28167332365135,250.07896920659033C70.17666984294237,320.27484674793965,64.84698225790005,396.69656628748305,111.28512138212992,450.4950937839243C156.20124167950087,502.5303643271138,231.32542653798444,500.4755392045468,300,503.46388370962813"></path>
                </svg>
                <i class="bi bi-scissors"></i>
              </div>
              <h4><a href="">Evolved Semantic Representations</a></h4>
              <p>The models returned by GP are the combinations of the semantic similarity scores of the three GO aspects, evolved to support PPI prediction.</p>
              <img src="assets/img/GPmodel-ppi.png" class="img-fluid" alt="">
            </div>
          </div>

        </div>

      </div>
  </section><!-- End Evaluation Section -->



<!-- ======= Results Section ======= -->
  <section id="results" class="results">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2> Results </h2>
          <div class="row justify-content-center">
            <div class="col-8" >
              <span>
              For evaluating the quality of a predicted classification, we use the weighted average of  <a href="https://en.wikipedia.org/wiki/F-score"> F-measures </a> (WAF). 
              Stratified 10-fold cross-validation was used for each experiment performed.  
              The results we report are the median of the 10 WAF values calculated on the 10 folds.
              </span>
            </div>
          </div>
        </div>

        <div class="row justify-content-center">
          <div class="col-4">
            <div id="datasetFilter_div"></div>
          </div>
          <div class="col-4">
            <div id="ssmFilter_div"></div>
          </div>
        </div>

        <div class="section-title">
          <div id="dashboard_div" >
            <div class="row">
              <div id="table_div" style="padding-top:30px;"></div>
            </div>
          </div>

          <style>
          .cssSelectedTableRow {
             font-size: 20px;
             font-weight:bold;
          }
          .cssHeaderCell {
             font-size: 16px;
             border: solid 1px #FFFFFF;
          }
          .cssTableCell {
             font-size: 16px;
             border: solid 1px #FFFFFF;
          }
          </style>

          <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
          <script type="text/javascript">
          google.charts.load('current', {'packages':['corechart', 'table', 'controls']});
          google.charts.setOnLoadCallback(drawMainDashboard);

          function drawMainDashboard() {

            var dashboard = new google.visualization.Dashboard(
            document.getElementById('dashboard_div'));

            var datasetFilter = new google.visualization.ControlWrapper({
            'controlType': 'CategoryFilter',
            'containerId': 'datasetFilter_div',
            'options': {
            'filterColumnIndex': 0,
            'ui': {
            'caption': 'All datasets',
            'labelStacking': 'horizontal',
            'label': 'Choose a PPI benchmark dataset:',
            'allowTyping': false,
            'allowMultiple': false
                  }
                },
            'state': {'selectedValues': ['ALL1']}
             });

            var ssmFilter = new google.visualization.ControlWrapper({
            'controlType': 'CategoryFilter',
            'containerId': 'ssmFilter_div',
            'options': {
            'filterColumnIndex': 1,
            'ui': {
            'caption': 'All measures',
            'labelStacking': 'horizontal',
            'label': 'Choose a similarity measure:',
            'allowTyping': false,
            'allowMultiple': false
                  }
                }
             });

            var cssClassNames = {
            'selectedTableRow': 'cssSelectedTableRow',
            'headerCell': 'cssHeaderCell',
            'tableCell': 'cssTableCell',
            };

            var table2 = new google.visualization.ChartWrapper({
            'chartType': 'Table',
            'containerId': 'table_div',
            'options': {'width': 760, 'height': 360, 'cssClassNames':cssClassNames},
            });

            var data2 = google.visualization.arrayToDataTable([
            ['Dataset', 'Measure', 'BP', 'CC', 'MF', 'Average', 'Maximum', 'evoKGsim+'],
            ['ALL1', 'ResnikMax + ICSeco'  ,0.760 ,0.713 ,0.646 ,0.749 ,0.743 ,0.765],
            ['ALL1', 'ResnikMax + ICResnik'  ,0.750 ,0.717 ,0.653 ,0.766 ,0.774 ,0.776],
            ['ALL1', 'ResnikBMA + ICSeco'  ,0.753 ,0.715 ,0.643 ,0.771 ,0.744 ,0.777],
            ['ALL1', 'ResnikBMA + ICResnik'  ,0.753 ,0.714 ,0.648 ,0.777 ,0.772 ,0.782],
            ['ALL1', 'SimGIC + ICSeco'  ,0.736 ,0.682 ,0.642 ,0.729 ,0.701 ,0.746],
            ['ALL1', 'SimGIC + ICResnik'  ,0.739 ,0.704 ,0.651 ,0.750 ,0.734 ,0.758],
            ['ALL1', 'TransE'  ,0.501 ,0.534 ,0.502 ,0.519 ,0.521 ,0.521],
            ['ALL1', 'distMult'  ,0.704 ,0.599 ,0.498 ,0.670 ,0.668 ,0.712],
            ['ALL1', 'RDF2Vec'  ,0.675 ,0.654 ,0.631 ,0.684 ,0.668 ,0.685],
            ['ALL1', 'OWL2Vec'  ,0.678 ,0.662 ,0.621 ,0.693 ,0.686 ,0.693],
            ['ALL3', 'ResnikMax + ICSeco'  ,0.747 ,0.685 ,0.630 ,0.735 ,0.719 ,0.751],
            ['ALL3', 'ResnikMax + ICResnik'  ,0.735 ,0.691 ,0.639 ,0.752 ,0.757 ,0.761],
            ['ALL3', 'ResnikBMA + ICSeco'  ,0.742 ,0.696 ,0.634 ,0.755 ,0.726 ,0.761],
            ['ALL3', 'ResnikBMA + ICResnik'  ,0.738 ,0.698 ,0.634 ,0.762 ,0.756 ,0.761],
            ['ALL3', 'SimGIC + ICSeco'  ,0.725 ,0.665 ,0.624 ,0.711 ,0.681 ,0.732],
            ['ALL3', 'SimGIC + ICResnik'  ,0.729 ,0.684 ,0.638 ,0.734 ,0.717 ,0.745],
            ['ALL3', 'TransE'  ,0.505 ,0.532 ,0.503 ,0.521 ,0.530 ,0.517],
            ['ALL3', 'distMult'  ,0.491 ,0.585 ,0.499 ,0.546 ,0.548 ,0.583],
            ['ALL3', 'RDF2Vec'  ,0.656 ,0.630 ,0.616 ,0.659 ,0.648 ,0.667],
            ['ALL3', 'OWL2Vec'  ,0.661 ,0.642 ,0.610 ,0.680 ,0.670 ,0.678],
            ['HS1', 'ResnikMax + ICSeco'  ,0.716 ,0.655 ,0.630 ,0.713 ,0.694 ,0.729],
            ['HS1', 'ResnikMax + ICResnik'  ,0.708 ,0.656 ,0.651 ,0.723 ,0.723 ,0.730],
            ['HS1', 'ResnikBMA + ICSeco'  ,0.711 ,0.678 ,0.634 ,0.729 ,0.700 ,0.735],
            ['HS1', 'ResnikBMA + ICResnik'  ,0.708 ,0.674 ,0.641 ,0.734 ,0.723 ,0.739],
            ['HS1', 'SimGIC + ICSeco'  ,0.695 ,0.662 ,0.626 ,0.697 ,0.671 ,0.712],
            ['HS1', 'SimGIC + ICResnik'  ,0.696 ,0.669 ,0.636 ,0.710 ,0.699 ,0.718],
            ['HS1', 'TransE'  ,0.502 ,0.509 ,0.537 ,0.530 ,0.532 ,0.530],
            ['HS1', 'distMult'  ,0.495 ,0.571 ,0.565 ,0.577 ,0.565 ,0.587],
            ['HS1', 'RDF2Vec'  ,0.651 ,0.611 ,0.606 ,0.652 ,0.641 ,0.661],
            ['HS1', 'OWL2Vec'  ,0.644 ,0.620 ,0.603 ,0.666 ,0.655 ,0.664],
            ['HS3', 'ResnikMax + ICSeco'  ,0.719 ,0.651 ,0.629 ,0.721 ,0.693 ,0.729],
            ['HS3', 'ResnikMax + ICResnik'  ,0.707 ,0.661 ,0.647 ,0.728 ,0.726 ,0.729],
            ['HS3', 'ResnikBMA + ICSeco'  ,0.715 ,0.679 ,0.637 ,0.732 ,0.701 ,0.741],
            ['HS3', 'ResnikBMA + ICResnik'  ,0.714 ,0.680 ,0.642 ,0.738 ,0.728 ,0.738],
            ['HS3', 'SimGIC + ICSeco'  ,0.702 ,0.657 ,0.626 ,0.699 ,0.675 ,0.716],
            ['HS3', 'SimGIC + ICResnik'  ,0.704 ,0.667 ,0.637 ,0.713 ,0.702 ,0.722],
            ['HS3', 'TransE'  ,0.503 ,0.505 ,0.539 ,0.525 ,0.537 ,0.532],
            ['HS3', 'distMult'  ,0.491 ,0.572 ,0.566 ,0.576 ,0.565 ,0.590],
            ['HS3', 'RDF2Vec'  ,0.650 ,0.617 ,0.607 ,0.657 ,0.642 ,0.657],
            ['HS3', 'OWL2Vec'  ,0.649 ,0.619 ,0.605 ,0.671 ,0.655 ,0.674],
            ['SC1', 'ResnikMax + ICSeco'  ,0.811 ,0.759 ,0.654 ,0.807 ,0.803 ,0.821],
            ['SC1', 'ResnikMax + ICResnik'  ,0.806 ,0.776 ,0.657 ,0.820 ,0.827 ,0.829],
            ['SC1', 'ResnikBMA + ICSeco'  ,0.796 ,0.752 ,0.651 ,0.809 ,0.793 ,0.815],
            ['SC1', 'ResnikBMA + ICResnik'  ,0.789 ,0.772 ,0.655 ,0.821 ,0.815 ,0.821],
            ['SC1', 'SimGIC + ICSeco'  ,0.782 ,0.717 ,0.668 ,0.773 ,0.750 ,0.794],
            ['SC1', 'SimGIC + ICResnik'  ,0.781 ,0.744 ,0.672 ,0.785 ,0.775 ,0.800],
            ['SC1', 'TransE'  ,0.538 ,0.530 ,0.525 ,0.557 ,0.536 ,0.544],
            ['SC1', 'distMult'  ,0.693 ,0.660 ,0.617 ,0.739 ,0.712 ,0.740],
            ['SC1', 'RDF2Vec'  ,0.756 ,0.725 ,0.672 ,0.768 ,0.744 ,0.774],
            ['SC1', 'OWL2Vec'  ,0.748 ,0.731 ,0.650 ,0.769 ,0.757 ,0.777],
            ['SC3', 'ResnikMax + ICSeco'  ,0.801 ,0.732 ,0.617 ,0.793 ,0.792 ,0.809],
            ['SC3', 'ResnikMax + ICResnik'  ,0.796 ,0.751 ,0.616 ,0.807 ,0.814 ,0.815],
            ['SC3', 'ResnikBMA + ICSeco'  ,0.789 ,0.733 ,0.633 ,0.799 ,0.777 ,0.807],
            ['SC3', 'ResnikBMA + ICResnik'  ,0.788 ,0.748 ,0.629 ,0.803 ,0.803 ,0.805],
            ['SC3', 'SimGIC + ICSeco'  ,0.770 ,0.691 ,0.626 ,0.753 ,0.727 ,0.779],
            ['SC3', 'SimGIC + ICResnik'  ,0.771 ,0.721 ,0.630 ,0.764 ,0.750 ,0.786],
            ['SC3', 'TransE'  ,0.553 ,0.536 ,0.520 ,0.565 ,0.552 ,0.562],
            ['SC3', 'distMult'  ,0.683 ,0.650 ,0.598 ,0.720 ,0.698 ,0.719],
            ['SC3', 'RDF2Vec'  ,0.732 ,0.696 ,0.644 ,0.736 ,0.711 ,0.748],
            ['SC3', 'OWL2Vec'  ,0.724 ,0.716 ,0.631 ,0.746 ,0.735 ,0.752],
            ['DM1', 'ResnikMax + ICSeco'  ,0.933 ,0.889 ,0.761 ,0.930 ,0.944 ,0.946],
            ['DM1', 'ResnikMax + ICResnik'  ,0.868 ,0.864 ,0.804 ,0.917 ,0.944 ,0.944],
            ['DM1', 'ResnikBMA + ICSeco'  ,0.921 ,0.889 ,0.804 ,0.960 ,0.919 ,0.958],
            ['DM1', 'ResnikBMA + ICResnik'  ,0.932 ,0.847 ,0.797 ,0.946 ,0.916 ,0.919],
            ['DM1', 'SimGIC + ICSeco'  ,0.930 ,0.860 ,0.769 ,0.889 ,0.861 ,0.917],
            ['DM1', 'SimGIC + ICResnik'  ,0.917 ,0.889 ,0.803 ,0.903 ,0.878 ,0.944],
            ['DM1', 'TransE'  ,0.551 ,0.526 ,0.526 ,0.563 ,0.477 ,0.509],
            ['DM1', 'distMult'  ,0.875 ,0.720 ,0.734 ,0.892 ,0.838 ,0.916],
            ['DM1', 'RDF2Vec'  ,0.831 ,0.837 ,0.783 ,0.874 ,0.878 ,0.889],
            ['DM1', 'OWL2Vec'  ,0.874 ,0.790 ,0.771 ,0.878 ,0.865 ,0.889],
            ['DM3', 'ResnikMax + ICSeco'  ,0.899 ,0.825 ,0.749 ,0.900 ,0.874 ,0.848],
            ['DM3', 'ResnikMax + ICResnik'  ,0.847 ,0.768 ,0.747 ,0.850 ,0.899 ,0.850],
            ['DM3', 'ResnikBMA + ICSeco'  ,0.900 ,0.825 ,0.749 ,0.950 ,0.874 ,0.900],
            ['DM3', 'ResnikBMA + ICResnik'  ,0.899 ,0.741 ,0.767 ,0.899 ,0.848 ,0.899],
            ['DM3', 'SimGIC + ICSeco'  ,0.850 ,0.850 ,0.798 ,0.874 ,0.825 ,0.874],
            ['DM3', 'SimGIC + ICResnik'  ,0.875 ,0.850 ,0.798 ,0.847 ,0.873 ,0.899],
            ['DM3', 'TransE'  ,0.495 ,0.490 ,0.566 ,0.572 ,0.522 ,0.507],
            ['DM3', 'distMult'  ,0.823 ,0.697 ,0.613 ,0.899 ,0.873 ,0.900],
            ['DM3', 'RDF2Vec'  ,0.775 ,0.798 ,0.747 ,0.799 ,0.796 ,0.873],
            ['DM3', 'OWL2Vec'  ,0.825 ,0.722 ,0.722 ,0.874 ,0.874 ,0.850],
            ['EC1', 'ResnikMax + ICSeco'  ,0.818 ,0.765 ,0.701 ,0.829 ,0.826 ,0.870],
            ['EC1', 'ResnikMax + ICResnik'  ,0.813 ,0.822 ,0.690 ,0.828 ,0.864 ,0.889],
            ['EC1', 'ResnikBMA + ICSeco'  ,0.777 ,0.794 ,0.741 ,0.824 ,0.817 ,0.856],
            ['EC1', 'ResnikBMA + ICResnik'  ,0.770 ,0.797 ,0.730 ,0.837 ,0.858 ,0.863],
            ['EC1', 'SimGIC + ICSeco'  ,0.792 ,0.758 ,0.693 ,0.836 ,0.808 ,0.817],
            ['EC1', 'SimGIC + ICResnik'  ,0.792 ,0.788 ,0.726 ,0.809 ,0.811 ,0.835],
            ['EC1', 'TransE'  ,0.510 ,0.498 ,0.454 ,0.520 ,0.483 ,0.503],
            ['EC1', 'distMult'  ,0.664 ,0.700 ,0.643 ,0.727 ,0.733 ,0.739],
            ['EC1', 'RDF2Vec'  ,0.767 ,0.746 ,0.702 ,0.842 ,0.770 ,0.833],
            ['EC1', 'OWL2Vec'  ,0.800 ,0.799 ,0.709 ,0.804 ,0.786 ,0.837],
            ['EC3', 'ResnikMax + ICSeco'  ,0.815 ,0.721 ,0.613 ,0.772 ,0.775 ,0.843],
            ['EC3', 'ResnikMax + ICResnik'  ,0.783 ,0.744 ,0.626 ,0.820 ,0.856 ,0.869],
            ['EC3', 'ResnikBMA + ICSeco'  ,0.777 ,0.809 ,0.648 ,0.832 ,0.821 ,0.809],
            ['EC3', 'ResnikBMA + ICResnik'  ,0.755 ,0.783 ,0.648 ,0.808 ,0.831 ,0.832],
            ['EC3', 'SimGIC + ICSeco'  ,0.749 ,0.772 ,0.607 ,0.808 ,0.752 ,0.808],
            ['EC3', 'SimGIC + ICResnik'  ,0.749 ,0.761 ,0.626 ,0.797 ,0.795 ,0.819],
            ['EC3', 'TransE'  ,0.517 ,0.510 ,0.483 ,0.569 ,0.513 ,0.543],
            ['EC3', 'distMult'  ,0.660 ,0.738 ,0.635 ,0.722 ,0.714 ,0.726],
            ['EC3', 'RDF2Vec'  ,0.731 ,0.748 ,0.671 ,0.820 ,0.785 ,0.809],
            ['EC3', 'OWL2Vec'  ,0.732 ,0.785 ,0.653 ,0.785 ,0.775 ,0.809]
            ]);

            dashboard.bind([datasetFilter, ssmFilter], [table2, table2]);
            dashboard.draw(data2);
            }

          </script>

      </div>

  </section><!-- End Results Section -->



 <!-- ======= Conclusions Section ======= -->
    <section id="conclusions" class="faq section-bg">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2> Conclusions </h2>
        </div>

         <div class="faq-list">

          <ul>
            <li data-aos="fade-up">
              <i class="bx bx-check-circle icon-help"></i> <a> evoKGsim+ with taxonomic similarity achieves a better performance than with embedding similarity </a> 
            </li>

            <li data-aos="fade-up" data-aos-delay="100">
              <i class="bx bx-check-circle icon-help"></i> <a> evoKGsim+ can generate tailored semantic representations that improve classification performance over static representations  </a>
            </li>

            <li data-aos="fade-up" data-aos-delay="200">
              <i class="bx bx-check-circle icon-help"></i> <a> evoKGsim+ can be readily generalized to other application and domains, where KG-based similarity is a suitable instance representation, such as prediction of drug-target interactions and gene-disease association, KG link prediction or recommendations.  </a>
            </li>

          </ul>
        </div>

      </div>
  </section><!-- End Conclusions Section -->



<!-- ======= About Video Section ======= -->
  <section id="minute-video" class="about-video">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>One-Minute Video</h2>
        </div>

        <div class="row justify-content-center">
          <img src="assets/img/OneMinuteVideo.png" class="img-fluid" alt="">
          <a href="https://www.youtube.com/watch?v=jDDaplaOz7Q" class="glightbox play-btn mb-4" data-vbtype="video" data-autoplay="true"></a>
        </div>

      </div>
  </section><!-- End About Video Section -->



<!-- ======= Authors Section ======= -->
  <section id="authors" class="team section-bg">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Authors</h2>
        </div>

        <div class="row justify-content-center">

          <div class="col-md-6 col-lg-3 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="fade-up" data-aos-delay="100">
            <div class="member">
              <div class="member-img">
                <img src="assets/img/authors/Rita.png" class="img-fluid" alt="">
                <div class="social">
                  <a href="https://scholar.google.com/citations?user=Mfnwam0AAAAJ&hl"><i class="bi bi-google"></i></a>
                  <a href="mailto:risousa@ciencias.ulisboa.pt"><i class="bi bi-envelope"></i></a>
                  <a href="https://twitter.com/RitaTorresSousa"><i class="bi bi-twitter"></i></a>
                </div>
              </div>
              <div class="member-info">
                <h4> Rita T. Sousa </h4>
                <span>LASIGE, Faculdade de CiÃªncias </span>
              </div>
            </div>
          </div>

          <div class="col-md-6 col-lg-3 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="fade-up" data-aos-delay="200">
            <div class="member">
              <div class="member-img">
                <img src="assets/img/authors/ProfSara.jpg" class="img-fluid" alt="">
                <div class="social">
                  <a href="https://scholar.google.com/citations?user=g15oqi0AAAAJ&hl"><i class="bi bi-google"></i></a>
                </div>
              </div>
              <div class="member-info">
                <h4> Sara Silva </h4>
                <span>LASIGE, Faculdade de CiÃªncias</span>
              </div>
            </div>
          </div>

          <div class="col-md-6 col-lg-3 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="fade-up" data-aos-delay="300">
            <div class="member">
              <div class="member-img">
                <img src="assets/img/authors/ProfCatia.jpg" class="img-fluid" alt="">
                <div class="social">
                  <a href="https://scholar.google.com/citations?user=8nGYegMAAAAJ&hl"><i class="bi bi-google"></i></a>
                  <a href="https://twitter.com/CPesquita"><i class="bi bi-twitter"></i></a>
                </div>
              </div>
              <div class="member-info">
                <h4> Catia Pesquita </h4>
                <span>LASIGE, Faculdade de CiÃªncias</span>
              </div>
            </div>
          </div>

        </div>

      </div>
  </section><!-- End Contact Section -->



<!-- ======= Funding Section ======= -->
  <section id="funding" class="funding">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2> Funding </h2>
          <p> Catia Pesquita, Sara Silva, Rita T. Sousa are funded by the FCT through LASIGE Research Unit, ref. UIDB/00408/2020 and ref. UIDP/00408/2020.
          Catia Pesquita and Rita T. Sousa are funded by project SMILAX (ref. PTDC/EEI-ESS/4633/2014), Sara Silva by projects BINDER (ref. PTDC/CCI-INF/29168/2017) 
          and PREDICT (ref. PTDC/CCI-CIF/29877/2017), and Rita T. Sousa by FCT PhD grant (ref. SFRH/BD/145377/2019).It was also partially supported by the KATY
          project which has received funding from the European Unionâs Horizon 2020 research and innovation programme under grant agreement No 101017453. </p> 
        </div>

        <div class="row justify-content-center">

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="100">
            <img src="assets/img/logos/ulisboa.png" class="img-fluid" alt="">
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="200">
            <img src="assets/img/logos/fcul.png" class="img-fluid" alt="">
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="300">
            <img src="assets/img/logos/fct.png" class="img-fluid" alt="">
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="400">
            <img src="assets/img/logos/lasige.jpg" class="img-fluid" alt="">
          </div>

        </div>

     </div>
  </section><!-- End Funding Section -->



  </main><!-- End #main -->



<!-- ======= Footer ======= -->
  <footer id="footer">

    <div class="footer-top">
      <div class="container">
        <div class="row"></div>
      </div>
    </div>

    <div class="container d-md-flex py-4">
      <div class="me-md-auto text-center text-md-start">
        <div class="credits">
          <!-- All the links in the footer should remain intact. -->
          <!-- You can delete the links only if you purchased the pro version. -->
          <!-- Licensing information: https://bootstrapmade.com/license/ -->
          <!-- Purchase the pro version with working PHP/AJAX contact form: https://bootstrapmade.com/onepage-multipurpose-bootstrap-template/ -->
          Designed by <a href="https://bootstrapmade.com/">BootstrapMade</a>
        </div>
      </div>
    </div>

  </footer>
<!-- End Footer -->

    <div id="preloader"></div>
    <a href="#" class="back-to-top d-flex align-items-center justify-content-center"><i class="bi bi-arrow-up-short"></i></a>

<!-- Vendor JS Files -->
  <script src="assets/vendor/aos/aos.js"></script>
  <script src="assets/vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
  <script src="assets/vendor/glightbox/js/glightbox.min.js"></script>
  <script src="assets/vendor/isotope-layout/isotope.pkgd.min.js"></script>
  <script src="assets/vendor/php-email-form/validate.js"></script>
  <script src="assets/vendor/purecounter/purecounter.js"></script>
  <script src="assets/vendor/swiper/swiper-bundle.min.js"></script>

<!-- Template Main JS File -->
<script src="assets/js/main.js"></script>

</body>

</html>
